admission_control/admission_control_proto/Cargo.toml

```diff
@@ -10,7 +10,7 @@ edition = "2018"
 futures = "0.1.28"
 futures03 = { version = "=0.3.0-alpha.17", package = "futures-preview" }
 grpcio = "0.4.3"
-protobuf = "2.7"
+protobuf = "~2.7"
```

Comment:
Why don't we bump v2.8 directly? Personally, I'm bit sceptical to allow version update flexibility on serialization, at least before we move to canonical serializers for transactions. Wdyt?

Reply:
This would require regenerating a lot of this serialization code + a breaking change which slows benchamarks. We could pin to "=2.7" but that would stop an update from 2.7.x to 2.7.y which would be a serialized bits compatible bugfix. This seems like a local optimal for now.

Reply:
Although convinced for now, I would prefer if we don't rely on compatible bugfix statements by the 3rd party devs for such critical Libra parts. A common strategy I've seen in the past is always posting a short review of affected lib changelog along with the version update commit, so we're 100% sure we've evaluated the changes by ourselves.